app/assets/stylesheets/profile.scss

```
@@ -206,24 +206,27 @@
         margin-top: 0;
       }
 
-      .logout-wrapper {
+      .profile-actions {
         display: inline-block;
         right: 0;
         position: absolute;
 
-        .logout {
-          background-color: $buttonColor;
-          color: $light-backgroundColor;
+        .button-static {
           font-size: $size-button-text;
           padding: 5px 20px;
           -moz-border-radius: 32px;
           -webkit-border-radius: 32px;
           -khtml-border-radius: 32px;
           border-radius: 32px;
+          font-weight: normal;
+          text-transform: none;
+
+          &.edit {
+            margin-right: 10px;
+          }
```

```suggestion
          margin: 0 5px;
```
Instead of setting a right margin on one button (which looks weird on mobile when the buttons wrap), set a consistent margin for all buttons.